09-intermediate-rails/consuming-apis-in-rails.md

```diff
@@ -148,15 +133,15 @@ def self.listchannels()
 end
 ```
 
-And finally, `sendmsg`:
+And finally, `send_msg`:
 
 ```ruby
-def self.sendmsg(channel, msg, token = nil)
+def self.send_msg(channel, msg, token = nil)
   token = TOKEN if token == nil
```

Comment:
Why do we have an argument for `token` here? I don't think this is ever used in the controller - we should omit it and use `TOKEN` everywhere.

If we want to keep it, we should say `token=TOKEN`, rather than having the line with the `if` statement.

Reply:
I will remove this - it was one of the leftovers that I was ignoring for no reason other than, I was.